src/main/java/io/strimzi/kafka/bridge/BridgeContentType.java

```diff
@@ -16,9 +16,16 @@ public class BridgeContentType {
     /** JSON encoding with BINARY embedded format */
     public static final String KAFKA_JSON_BINARY = "application/vnd.kafka.binary.v2+json";
 
+
+    /** JSON encoding with Plain text embedded format */
+    public static final String KAFKA_JSON_TEXT = "application/vnd.kafka.string.v2+json";
```

Comment:
@ppatierno just having a look at this, I think it can be one of the reason.
while retrieving the messages, I was using `application/vnd.kafka.text.v2+json`.

Reply:
so you should change here from `string` to `text` then.

Reply:
Yes,I have changed that!
I am getting a new error now, 
``json
{"offsets":[{"partition":0,"offset":0},{"error_code":404,"message":"Topic bridge-quickstart-topic not present in metadata after 60000 ms."},{"partition":0,"offset":1}]}
```
I was already getting this, and I had to use kRafta instead of Zookeeper. Now even using kRafta, I am still facing the issue while producing messages!
any thought?

Reply:
Maybe you should create the topic first? Anyway I think you have more problems with this PR ... I played with it and going to leave some feedback soon.

Reply:
> Maybe you should create the topic first? Anyway I think you have more problems with this PR ... I played with it and going to leave some feedback soon.

I have created the topic!

Reply:
@ppatierno this error persists. Seems that the data got produced, but I don't get the expected response from the HTTP bridge.

Reply:
@ppatierno I finally got rid of this.
The problem happens if I try to produce messages to a partition that does not exist in the topic.

My topic is configured like

```
Topic: bridge-quickstart-topic	TopicId: MY83JIVaReuXx2-xvKdcWg	PartitionCount: 1	ReplicationFactor: 1	Configs: segment.bytes=1073741824
	Topic: bridge-quickstart-topic	Partition: 0	Leader: 1	Replicas: 1	Isr: 1
```
We have `partition:0`, if I try to produce message like

```json
  {
      "value": "sales-lead-0002",
      "partition": 1
  }
```
the following error will occur
```json
  {
            "error_code": 404,
            "message": "Topic bridge-quickstart-topic not present in metadata after 60000 ms."
 }
```

After changing to partition:0 or not mentioning the partition, the error fade away.